build.gradle

```diff
@@ -20,41 +20,54 @@
 
 import groovy.json.JsonBuilder
 import groovy.json.JsonSlurper
+import org.apache.tools.ant.taskdefs.condition.Os
 
 buildscript {
     repositories {
         maven {
-            url "${repositoryURL}/repo"
+            url "${repositoryURL}/artifactory/maven-onprem-dev"
```

Comment:
Not sure why we expose this now.
I suspect gradle isn't needed at all outside of our internal build system. So, the open source aspect of this product is not affected by this, but I would be happier if it was not published to public github.

Reply:
I'll create a ticket to remove gradle from App Browser for 4.05. Can we release as it is for now?

Reply:
I just removed gradle.